### PR TITLE
updates for helm chart

### PR DESCRIPTION
**Pull request by liyaka**
## Issues

Resolves #

### CODE REVIEW

File: .github/workflows/build_and_push_docker.yaml [hunk 1]
```diff
@@ -26,6 +26,12 @@ on:
             required: false
             default: ""
             description: Arguments for cloud docker build
+        push_latest:
+            type: string
+            required: true
+            description: If to push latest
```
Comment (andrescrz) [inline .github/workflows/build_and_push_docker.yaml]: Minor: description hard to understand, please update.

File: .github/workflows/build_apps.yml [hunk 2]
```diff
@@ -62,7 +63,12 @@ jobs:
             else
               echo "build_from=${{inputs.version}}" | tee -a $GITHUB_OUTPUT
             fi
-            echo "Build from: "
+            if [[ "${{inputs.version}}" == "" || "${{ github.ref_name }}" == "liya/DEV-2307-updates-for-helm-chart" ]]; then
```
Comment (andrescrz) [inline .github/workflows/build_apps.yml]: Why there's a reference to your branch?

**Review by andrescrz — APPROVED** [review]
Left a couple of comments with a question and a minor one. But no blockers.

File: .github/workflows/build_apps.yml (hunk 2 above)
Comment (liyaka) [reply .github/workflows/build_apps.yml]: Thanks :) need to change to main, was set to my branch for tests..

**Review by andrescrz — APPROVED** [review]

**Pull request merged**